README.md

```diff
@@ -108,9 +108,17 @@ Connect your docker registry with minikube with:
 
 Then run:
 ```
-harness-deployment cloud-harness . -l  -n osblocal -d osb.local -u -dtls -m build -e local -i osb-portal 
+harness-deployment cloud-harness . -l  -n osblocal -d osb.local -u -dtls -e local -i osb-portal
 ```
 You do not need to run the port-forwarding commands on the local deployment.
+
+Here, you can modify the argument of the `-e` option to select what environment you want to deploy.
+These correspond to the files from the `osb-portal/deploy` directory.
```

Comment:
Mainly that file but not just that file. Any values.yaml file can have a values-[ENV].yaml file

Reply:
Done (better late than never :sweat_smile: )